workbench-core/accounts/src/models/account.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { HostingAccountStatus } from '../constants/hostingAccountStatus';
+
+interface Account {
```

Comment:
Is this supposed to be a copy of [this](https://github.com/aws-solutions/solution-spark-on-aws/blob/b86655189e529a5d7c6f0afb346c4efcf8af4712/workbench-core/accounts/src/services/accountService.ts#L19). I think there's some differences there.

`CC` should be included and `cidr` shouldn't be there.

Reply:
Will be fixed on the feature branch.